src/commands/generate/__tests__/resources/test-cases/ete-binary/product/eteBinaryService.ts

```diff
@@ -1,9 +1,9 @@
 import { IHttpApiBridge, MediaType } from "conjure-client";
 
 export interface IEteBinaryService {
-    postBinary(body: any): Promise<any>;
-    getOptionalBinaryPresent(): Promise<any | null>;
-    getOptionalBinaryEmpty(): Promise<any | null>;
+    postBinary(body: Blob): Promise<Blob>;
+    getOptionalBinaryPresent(): Promise<Blob | null>;
+    getOptionalBinaryEmpty(): Promise<Blob | null>;
```

Comment:
Looks like a bonus fix for #88 as well

Reply:
Right, didn't notice that before. Happy to split this into two prs if maintainers prefer it

Reply:
I think it makes sense to fix that in this PR, the problem is very similar.